### PR TITLE
implement output format html for cocoa widgetset

### PR DESCRIPTION
**Pull request by charlesknightsbridge**
Requires components from LCLWebBrowser which implement direct calls to MacOS api for HTML display; uses native handlers for copying html on screen

### CODE REVIEW

**Comment by torstenchr** [comment]
All files/folders name .DS_Store are mac specific and should not be part of the commit.

**Comment by charlesknightsbridge** [comment]
I’ll try to skip them next time. I’ll see if there is a setting in GitHub Desktop to skip hidden files.

Jamie

**Comment by torstenchr** [comment]
I have removed the folders in question such that they will not be part of the final merge. Please make sure to do a "git pull" before any further work on the branch.

**Comment by charlesknightsbridge** [comment]
Would not merge because of differences in .gitignore! That's been resolved, so over to you.

**Review by torstenchr — CHANGES_REQUESTED** [review]
Once the .gitignore file has been reverted back to before the PR is ready for approval.

File: .gitignore [hunk 1]
```diff
@@ -7,5 +7,3 @@ target/
 *.resolved
 *.lps
 commandlog.pgm
-src/grammar/*.cgt
```
[the remark this answers is not in the document]
Comment (charlesknightsbridge) [reply .gitignore]: It's mystery to me...no doubt my fault, but done by the github app. I forget that .gitignore is in the repository.

**Comment by charlesknightsbridge** [comment]
Change committed; back to you, Torsten. I can't approve and it won't go automatically.

**Review by torstenchr — APPROVED** [review]
Perfect!

**Pull request merged**